symengine/printer.h

```diff
@@ -59,7 +59,7 @@ class Precedence : public BaseVisitor<Precedence> {
             if(x.imaginary_ == 1) {
                 precedence = PrecedenceEnum::Atom;
             } else {
-                precedence = PrecedenceEnum::Mul;
+                precedence = PrecedenceEnum::Add;
```

Comment:
Is this a bug in the `Precedence` class? What is the rationale behind this change?